engine/src/main/java/org/terasology/engine/module/StandardModuleExtension.java

```diff
@@ -47,17 +59,34 @@ public String getKey() {
     }
 
     public static boolean isServerSideOnly(Module module) {
-        Boolean serverSideOnly = module.getMetadata().getExtension(SERVER_SIDE_ONLY.getKey(), Boolean.class);
-        return serverSideOnly != null && serverSideOnly;
+        return getBooleanExtension(module, SERVER_SIDE_ONLY);
     }
 
     public static boolean isGameplayModule(Module module) {
-        Boolean isGameplay = module.getMetadata().getExtension(IS_GAMEPLAY.getKey(), Boolean.class);
-        return isGameplay != null && isGameplay;
+        return getBooleanExtension(module, IS_GAMEPLAY);
     }
 
```

Comment:
I dropped the other is*Module() methods which @TheFlash98 added since they aren't used anywhere yet.   Could either add them back in when they are actually used, or just leave them out.